ocaml-lsp-server/src/configuration.ml

```diff
@@ -14,28 +17,38 @@ let default () =
     in
     Lev_fiber.Timer.Wheel.create ~delay
   in
-  { wheel }
+  let data =
+    Config_data.
+      { codelens = Some { enable = Some true }
```

Comment:
Moving creation of defaults to `Config_data` module would make the structure better. That way, we would have them closer to type definition. Now we are inlining the defaults, which is harder to read. I think it's better to call here `Config_data.default` instead

Reply:
Done 175d5e64a0df1d98ad4b31a3a7b5509973d8b554